lmfdb/classical_modular_forms/web_newform.py

```diff
@@ -954,6 +954,55 @@ def td_wrap(val):
         twists.extend(['</tbody>', '</table>'])
         return '\n'.join(twists)
 
+    def display_hecke_char_polys(self, num_disp = 5):
+      # Display a table of the characteristic polynomials of the Hecke operators for small primes
+      # Right now, the number of primes presented by default is 5, but that could be changed easily
+      # The rest could be seen by using "show more" / "show less" - 
+      # The code for the table wrapping, scrolling etc. is common with many others and should be eventually 
+      # replaced by a call to a single class/function with some parameters
+        def th_wrap(kwl, title):
+            return '    <th>%s</th>' % display_knowl(kwl, title=title)
+        def td_wrap(val):
+            return '    <td>$%s$</th>' % val
+        data = db.mf_newforms.lookup(self.label)
+        hecke_orbit_code = data['hecke_orbit_code']
+        self.heckepolys = []
+        R = PolynomialRing(ZZ, 'T')
+        T = R.gen(0)
```

Comment:
This line needs to be deleted as the variable `T` is never used.

Reply:
Oops, sorry. Also noticed that I should add the change to new spaces as well. On it.